packages/builder/src/components/automation/SetupPanel/AutomationBlockSetup.svelte

```diff
@@ -140,6 +144,7 @@
       ? [hbAutocomplete([...bindingsToCompletions(bindings, codeMode)])]
       : []
 
+  // TODO: check if it inputData != newInputData (memo)
```

Comment:
Not sure if these are done and left in by accident, or still need done, so just flagging them!